Amplify/Core/Configuration/Internal/Amplify+Reset.swift

```diff
@@ -22,7 +22,12 @@ extension Amplify {
 
         let group = DispatchGroup()
 
-        for categoryType in CategoryType.allCases {
+        let manuallyResetCategories = [CategoryType.logging, .hub]
```

Comment:
nit: `let manuallyResetCategories: [CategoryType] = [.logging, .hub]`

Reply:
Synced offline. You can take a look again.